src/main/java/frc/robot/RobotContainer.java

```diff
@@ -142,13 +141,13 @@ private void configureButtonBindings() {
 
     intake.whenPressed(new InstantCommand(() -> {
       m_intake.setExtended(true);
-    }, m_intake));
+    }));
 
     intake.whenReleased(new InstantCommand(() -> {
       m_intake.setExtended(false);
       m_intake.stop();
       m_revolver.stop();
-    }, m_intake, m_revolver));
+    }));
```

Comment:
This stop logic shouldn't be here, it's unrelated to retracting the intake. You should create full commands for the intake roller & revolver logic, so that you can add in the stop behaviours there.